modules/update-evaluate-availability.adoc

```diff
@@ -0,0 +1,114 @@
+If the CVO finds that the cluster doesn't match the risks of an update, or that there are no risks associated with the update, it stores the target version in the `availableUpdates` field of its ClusterVersion resource.
+
+The user interface, either the web console or the OpenShift CLI (`oc`), then presents this information to the administrator.
+Each "supported but not recommended" update recommendation contains a link to further resources about the risk so that the administrator can make an informed decision about the update.
+
+You can inspect all available updates with the following command:
+
+[source,terminal]
+----
+$ oc adm upgrade --include-not-recommended
+----
+
+The additional `--include-not-recommended` parameter includes updates that are available but not recommended due to a known risk that applies to the cluster.
+
+.Example output
+[source,terminal]
+----
+Cluster version is 4.10.22
+
+Upstream is unset, so the cluster will use an appropriate default.
+Channel: fast-4.11 (available channels: candidate-4.10, candidate-4.11, eus-4.10, fast-4.10, fast-4.11, stable-4.10)
+
+Recommended updates:
+
+  VERSION     IMAGE
+  4.10.26     quay.io/openshift-release-dev/ocp-release@sha256:e1fa1f513068082d97d78be643c369398b0e6820afab708d26acda2262940954
+  4.10.25     quay.io/openshift-release-dev/ocp-release@sha256:ed84fb3fbe026b3bbb4a2637ddd874452ac49c6ead1e15675f257e28664879cc
+  4.10.24     quay.io/openshift-release-dev/ocp-release@sha256:aab51636460b5a9757b736a29bc92ada6e6e6282e46b06e6fd483063d590d62a
+  4.10.23     quay.io/openshift-release-dev/ocp-release@sha256:e40e49d722cb36a95fa1c03002942b967ccbd7d68de10e003f0baa69abad457b
+
+Supported but not recommended updates:
+
+  Version: 4.11.0
+  Image: quay.io/openshift-release-dev/ocp-release@sha256:300bce8246cf880e792e106607925de0a404484637627edf5f517375517d54a4
+  Recommended: False
+  Reason: RPMOSTreeTimeout
+  Message: Nodes with substantial numbers of containers and CPU contention may not reconcile machine configuration https://bugzilla.redhat.com/show_bug.cgi?id=2111817#c22
+----
+
+One way to inspect the underlying availability data created by the CVO is by querying the `ClusterVersion` resource with the following command:
+
+[source,terminal]
+----
+$ oc get clusterversion version -o json | jq '.status.availableUpdates'
+----
+
+.Example output
+[source,terminal]
+----
+[
+  {
+    "channels": [
+      "candidate-4.11",
+      "candidate-4.12",
+      "fast-4.11",
+      "fast-4.12"
+    ],
+    "image": "quay.io/openshift-release-dev/ocp-release@sha256:400267c7f4e61c6bfa0a59571467e8bd85c9188e442cbd820cc8263809be3775",
+    "url": "https://access.redhat.com/errata/RHBA-2023:3213",
+    "version": "4.11.41"
+  },
+  ...
+]
+----
+
+The same command can be used to check conditional updates:
```

Comment:
Is is _really_ the same command if one of the flags is different? 

```suggestion
You can use the same command to check conditional updates, as shown in the following example:
```

Reply:
yeah not exactly the same since we're accessing different json path with jq. perhaps s/the same/a similar

Reply:
Sounds good, I can use the following wording: "A similar command can be used..."